builtin/providers/google/resource_dns_managed_zone.go

```diff
@@ -32,7 +34,6 @@ func resourceDnsManagedZone() *schema.Resource {
 				Type:     schema.TypeString,
 				Optional: true,
 				ForceNew: true,
-				Default:  "Managed by Terraform",
```

Comment:
My only concern about this is that removing the default would lead Terraform to think every managed zone changed, and try to destroy then re-create it. I'm not sure what the best way to get rid of a default value is (or support imports when dealing with a default value) without causing too much backwards incompatibility. I'll check with the team and see if anyone has advice and report back.